e2e/app/element/base-element.ts

```diff
@@ -187,6 +182,7 @@ export default class BaseElement extends Container {
     const typeAndCheck = async () => {
       const actualValue = await clearAndType(textValue);
       if (actualValue === textValue) {
+        await this.pressTab();
```

Comment:
some input fields do not validate entered value unless mouse tabs out of textbox/input.